controllers/app.controllers.js

```diff
@@ -15,6 +19,16 @@ exports.getTopics = (req, res, next) => {
     });
 };
 
+exports.getArticles = (req, res, next) => {
+  selectArticles()
+    .then((articles) => {
+      res.status(200).send(articles);
```

Comment:
You should send articles on an object - then in your test you can access the articles on body.articles for example. It makes more sense than just on body!